files/default/event_handler/write_chef_error_handler.rb

```diff
@@ -0,0 +1,14 @@
+
+module WriteChefError
+  class WriteChefError < Chef::Handler
+    def report
+      if run_status.failed?
+        error_file = '/var/log/parallelcluster/bootstrap_error_msg'
```

Comment:
this error_file should be defined in the default/attributes.rb file and we should also define a variable for `/var/log/parallelcluster/` and use it everywhere.